### PR TITLE
Initial draft for TWIT October 2016

### PR DESCRIPTION
**Pull request by HipsterBrown**
Let's update the community on the latest and greatest in Tessel-land!

Looking for some review @tcr, @Frijol, and @rwaldron.

### CODE REVIEW

File: 2016-10-14-TWIT.md
Comment (tcr) [inline 2016-10-14-TWIT.md]: Make "Rust programs" link to [tessel-rust](http://github.com/tessel/tessel-rust)

File: 2016-10-14-TWIT.md
Comment (tcr) [inline 2016-10-14-TWIT.md]: "for folks who want to contribute to open source and get involved with Tessel."

File: 2016-10-14-TWIT.md
Comment (tcr) [inline 2016-10-14-TWIT.md]:
```
As we iterating on the [hardware and module APIs for Rust](https://github.com/tessel/tessel-rust), we wanted to further improve the experience for Rust developers. Tim Ryan ([@trimryan](https://twitter.com/trimryan)) has added support for local compilation of Rust programs in the latest release of [t2-cli](https://github.com/tessel/t2-cli/releases/tag/v0.1.0). You can now cross-compile Rust code for an embedded device seamlessly using `t2 run`. Check out [the repo](http://github.com/tessel/tessel-rust) for instructions, try it out, and let us know how it's working for you in the #rust-lang channel of our [Slack community](https://tessel-slack.herokuapp.com/).
```

File: 2016-10-14-TWIT.md
Comment (tcr) [inline 2016-10-14-TWIT.md]: Wow, 10 whole days? that's insane

**Review by tcr — CHANGES_REQUESTED** [review]

File: 2016-10-14-TWIT.md
Comment (Frijol) [inline 2016-10-14-TWIT.md]: Love the enthusiasm, but probably shouldn't have every single one end with ! ;)

File: 2016-10-14-TWIT.md
Comment (Frijol) [inline 2016-10-14-TWIT.md]: "As part of our goals this year to increase contributions and encourage both inclusion and accessibility to newcomers, we've labeled [a bunch of issues](https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+label%3AHacktoberFest+user%3Atessel+) for folks who want to get started!" --> "We've labeled [a bunch of issues](https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+label%3AHacktoberFest+user%3Atessel+) for folks who want to get started! Since one of our two [core goals this year](https://github.com/tessel/project#current-goals) is to increase contributions and encourage both inclusion and accessibility to newcomers, we are proud to be a part of this project."

..or something like that

File: 2016-10-14-TWIT.md
Comment (Frijol) [reply 2016-10-14-TWIT.md]: "As we iterating on" --> "As we iterate on"

File: 2016-10-14-TWIT.md
Comment (Frijol) [reply 2016-10-14-TWIT.md]: We won't actually be there all 10 days tho

File: 2016-10-14-TWIT.md
Comment (Frijol) [inline 2016-10-14-TWIT.md]: Needs its own heading, not a part of the prior one

File: 2016-10-14-TWIT.md
Comment (Frijol) [inline 2016-10-14-TWIT.md]: would phrase more "we are investigating as the potential brains"

File: 2016-10-14-TWIT.md
Comment (Frijol) [inline 2016-10-14-TWIT.md]: Cool!

**Comment by Frijol** [comment]
Thanks for writing this! I didn't do the official review process b/c don't want to hold you up waiting for an approval on requested changes

**Review by tcr — APPROVED** [review]

**Pull request merged**
